src/main2.cpp

```diff
@@ -793,54 +634,60 @@ int main(int argc, char* argv[])
 			// Clear depth data on the frame buffer
 			glClear(GL_DEPTH_BUFFER_BIT);
 
-			//Note .8 != 30 for the height so that we can visually see the affects of rotating
+            // Update position of light
 			updateLight(glm::vec3(x, lightDepth, z), glm::vec3(0, 0, 0), SceneObj, shaderProgram, i, true);
 			if (i == 1.0f) i = -1.0f;
 
+            // DRAW GEOMETRY
+            // Draw scoreboard
 			numberDraw.Scoreboard(blueScore,false,true);
 			numberDraw2.Scoreboard(redScore,false,false);
 
+            // Draw blue player and racket
 			playerArm1.SetAttr(groupMatrix, renderAs, shaderProgram);
 			playerArm1.DrawArm();
 			racket1.SetAttr(groupMatrix, renderAs, shaderProgram, playerArm1.partParent);
 			racket1.Draw();
 
+            // Draw red player and racket
             playerArm2.SetAttr(groupMatrix, renderAs, shaderProgram);
             playerArm2.DrawArm();
             racket2.SetAttr(groupMatrix, renderAs, shaderProgram, playerArm2.partParent);
             racket2.Draw();
+
+            // Draw ball
 			ball.setGroupMatrix(groupMatrix);
 			ball.setRenderAs(renderAs);
 			ball.drawBall();
 
+            // Draw court and skybox
             SceneObj.sphereVao = unitSphereAO;
             SceneObj.sphereVertCount = (int) vertexIndicesSphere.size();
             SceneObj.SetAttr(rotationMatrixW, renderAs, shaderProgram);
             SceneObj.SetVAO(unitCubeAO, gridAO);
             SceneObj.DrawScene(false);  // Draw scene without the skybox, so it can't be used to make shadows on the scene
 
-			//This is all for bleacher			
+			// This is all for the bleachers
```

Comment:
ideally bleachers and red court could get combined into scene objects

Reply:
Perhaps. I can try that